stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/list/adapter/view/internal/MediaAttachmentView.kt

```diff
@@ -139,7 +168,40 @@ internal class MediaAttachmentView : ConstraintLayout {
         }
     }
 
+    private fun Attachment.giphyInfo(field: GiphyInfoType): GiphyInfo? {
```

Comment:
It would be really nice to do parsing earlier and put those parameters directly in the `Attachment` object but we would have to make them nullable. Maybe `Attachment` model can be improved in the future